source/before-integrating/integrating-third-party-platform.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: Integrating third-party platforms with GOV.UK One Login
+weight: 1
+last_reviewed_on: 2024-07-04
+review_in: 6 months
+---
+
+
+# Integrating third-party platforms with GOV.UK One Login
+
+If you use a third-party platform (Software as a Service such as Salesforce or Microsoft Power Platform, or an identity provider such as Amazon Cognito or ForgeRock) to integrate with GOV.UK One Login, you might experience issues or specific limitations during integration.
+
+Contact GOV.UK One Login at govuk-one-login@digital.cabinet-office.gov.uk ([govuk-one-login@digital.cabinet-office.gov.uk](mailto:govuk-one-login@digital.cabinet-office.gov.uk) will open a link to your mail client) if you’re using a third-party platform to integrate with GOV.UK One Login. 
+
+GOV.UK One Login will update this page regularly with detailed information on integrating with third-party platforms
```

Comment:
```suggestion
GOV.UK One Login will update this page regularly with information on integrating with third-party platforms.
```
Is this sentence helpful to users? We're not saying what 'regularly' is and the page itself already sets the expectation we'll provide information about integrating with third-party platforms!

Reply:
good point ... will amend @sabrinaharris